src/pybind/mgr/orchestrator.py

```diff
@@ -718,55 +763,70 @@ def validate(self, all_hosts):
 
 
 class StatelessServiceSpec(object):
-    # Request to orchestrator for a group of stateless services
-    # such as MDS, RGW, nfs gateway, iscsi gateway
     """
     Details of stateless service creation.
 
-    This is *not* supposed to contain all the configuration
-    of the services: it's just supposed to be enough information to
-    execute the binaries.
+    Request to orchestrator for a group of stateless services
+    such as MDS, RGW or iscsi gateway
     """
+    # This structure is supposed to be enough information to
+    # start the services.
 
-    def __init__(self):
-        self.placement = PlacementSpec()
+    def __init__(self, name, placement=None, count=None):
```

Comment:
```suggestion
    def __init__(self, name, placement=None, count=1):
```

Reply:
I'm not feeling well with defining any default. I'd rather leave the defaults to the orchestrators.